utils/diff.py

```diff
@@ -22,7 +41,23 @@ def diff(infile1, infile2, ignore_unassigned=False):
         elif old_asn == 0:
             print("%s AS%i # was unassigned" % (net, new_asn))
         else:
-            print("%s AS%i # was AS%i" % (net, new_asn, old_asn))
+            if len(ips) > 0:
+                if old_asn not in old_asn_changed:
+                    old_asn_changed.append(old_asn)
+                    for ip in ips:
+                        ip_type = "IPv4" if type(ipaddress.ip_address(ip)) is ipaddress.IPv4Address else "IPv6"
+                        network_type = "IPv4" if type(ipaddress.ip_network(net)) is ipaddress.IPv4Network else "IPv6"
+                        if ip_type == "IPv4" and network_type == "IPv4":
+                            if ipaddress.IPv4Address(ip) in ipaddress.IPv4Network(net):
+                                print(f"{old_asn} -> {new_asn}")
```

Comment:
One will want to de-duplicate these entries (e.g., show `AS123 -> AS345` just once). This could be done with bash of course, but you might wanna do it here.

Or maybe like this:
"AS123 -> AS345": [ip1, ip2, ip3]
"AS2 -> AS345": [ip4, ip5]

Reply:
It shows just once. See that there is a `break` right after printing {old_asn} -> {new_asn}.